pkgs/development/compilers/cudatoolkit/redist/build-cuda-redist-package.nix

```diff
@@ -29,7 +31,7 @@ in stdenv.mkDerivation {
   ];
 
   buildInputs = [
-    stdenv.cc.cc.lib
+    cudatoolkit.cc.cc.lib
```

Comment:
Hypothesis: the [``GLIBCXX_3.4.30' not found`](https://hercules-ci.com/accounts/github/SomeoneSerge/derivations/%2Fnix%2Fstore%2F3iaz2y7b0pgbj4f8y4cim6ldajr5334i-tensorflow-gpu-2.11.0.drv/log?via-job=6aab21a5-3e37-4365-b4e2-9fe6a1ca5f21) errors in e.g. tensorflow had to do with us using `libstdc++` from `stdenv`. I didn't try building without this change simply because it takes compute. Anyone is welcome to try and do that

Reply:
Invalidated: https://gist.github.com/SomeoneSerge/02419e1534499948d74f4d68db5d7bea

Reply:
I consider still keeping this change:
- It's strange we weren't linking cuda against the "officially compatible" `libstdc++` in the first place.
- I think, any downstream package that uses `nvcc` ends up bringing in `gcc11` and its libraries into the closure anyway